### PR TITLE
Development

### PR DESCRIPTION
**Pull request by pradyu1**
Updated code, please review it 
@davidcavazos @azizketari

### CODE REVIEW

File: pdf-converter/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+This is NOT an official Google product.
+
+# Prepare for installing
```
Comment (davidcavazos) [inline pdf-converter/README.md]: Can you make this into a `Dockerfile` so we can create the CE instance using that container image?

Comment (davidcavazos) [inline pdf-converter/README.md:2]: Can you make this into a `Dockerfile` so we can create the CE instance using that container image?

**Review by davidcavazos — COMMENTED** [review]
Everything looks good, we're just missing having the Compute Engine instance use a Docker image rather than having to configure it manually

File: pdf-converter/README.md (hunk 1 above)
Comment (pshaikhOfficial) [reply pdf-converter/README.md:2]: @davidcavazos  Created Dockerfile and deployed on CE as the container. Updated README.md file as well.

Comment (davidcavazos) [reply pdf-converter/README.md:2]: Thank you! Merging

**Pull request merged**